_playwright-tests/UI/Templates.spec.ts

```diff
@@ -8,11 +8,27 @@ test.describe('Templates', () => {
   }) => {
     await navigateToTemplates(page);
     await closePopupsIfExist(page);
-
-    const AddButton = page.locator('[data-ouia-component-id="create_content_template"]');
-
-    // Wait for the Add button to become enabled (up to 10 seconds)
-    await AddButton.first().isEnabled({ timeout: 10000 });
+    await page.getByRole('button', { name: 'Add content template' }).click();
+    await page.getByRole('button', { name: 'Select architecture' }).click();
+    await page.getByRole('option', { name: 'x86_64' }).click();
+    await page.getByRole('button', { name: 'Select version' }).click();
+    await page.getByRole('option', { name: 'el8' }).click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Add the step to select 'Red hat repos Checkbox'
+    await page.getByRole('button', { name: 'Next' }).click();
```

Comment:
keep in mind that in our CI there won't be any repos other than the "Ansible Engine for RHEL 8" which will not be included by default and you will actually need to click the checkbox for it, otherwise it won't let you continue to the next step 😅 
so that needs to be fixed 🔧